etc/ciao/ciao.json

```diff
@@ -0,0 +1,9 @@
+{
+    "server": "localhost3",
+    "cacert": "/etc/pki/ciao/CAcert-server-localhost.pem",
```

Comment:
Real ciao-cert generated CA certificates are going to look like: CAcert-ciao-scheduler.example.com.pem (To match my previous comment). Please use that.